src/tests/admins.test.js

```diff
@@ -1 +1,122 @@
-test.skip('skip', () => {});
+import request from 'supertest';
+import app from '../app';
+import Admin from '../models/admins';
+import adminSeed from '../seeds/admins';
+
+const mockAdmin = {
+  firstName: 'Admin',
+  lastName: 'Admin',
+  dni: 61165476,
+  phone: 1234567890,
+  email: 'adminPost@admin.com',
+  city: 'Admin',
+  password: 'Admin1234',
+};
+const mockAdminEmail = {
+  firstName: 'Admin',
+  lastName: 'Admin',
+  dni: 51165476,
+  phone: 1234567890,
+  email: 'adminPost@admin.com',
+  city: 'Admin',
+  password: 'Admin1234',
+};
+const mockAdminDni = {
+  firstName: 'Admin',
+  lastName: 'Admin',
+  dni: 41165476,
+  phone: 1234567890,
+  email: 'adminPost2@admin.com',
+  city: 'Admin',
+  password: 'Admin1234',
+};
+
+beforeAll(async () => {
+  await Admin.collection.insertMany(adminSeed);
+});
+
+describe('GET /api/admins', () => {
+  test('should return status 200 and length to be 2', async () => {
+    const response = await request(app).get('/api/admins').send();
+    expect(response.body.data.length).toBeGreaterThan(0);
+    expect(response.body.data.length).toBe(2);
```

Comment:
I don't know if it's okay this test,  if the database is updated this test we're going to fail

Reply:
but the test DB exist only to do this test, with the beforeAll i load only 2 admins